.gitignore

```diff
@@ -58,3 +58,4 @@ __pycache__/
 dmm-tools.exe
 OpenDream
 paradise.json
+$RECYCLE.BIN
```

Comment:
I couldn't not add this as my dev environment is now within a windows dev drive root, won't cause any harm